test/bin/sandman/test_sweep.py

```diff
@@ -382,8 +388,13 @@ def test_tracked_file_stashed(self, vault_mock):
     # Behavior: When a regular, untracked, non-vault file has been there for
     # more than the deletion threshold, and it has been notifed to somebody,
     # the source is deleted and a hardlink created in Limbo
+    @mock.patch.dict(os.environ, {
+        "SANDMANRC": f"{_HGI_FARM_SANDMAN_CONFIG_LOCATION}/sandmanrc.{getpass.getuser()}"
```

Comment:
Likewise, can we have a function defined in one place that returns the correct value here, to be re-used throughout this file and in test_persistence.py?